src/anim/anim.js

```diff
@@ -640,6 +640,23 @@ function DefaultAnimBinder(graph) {
     };
     flatten(graph);
 
+    var findMeshInstances = function (node) {
+        var object = node;
+        while (object && object.constructor !== Entity) {
```

Comment:
I think the preferred way to check this is ``object instanceof Entity`

Reply:
I'll fix. @ellthompson - I grabbed this line from anim.js, maybe improve it there as well.

Reply:
fair point, I only refactored existing code, this was there. I'll change it.

Reply:
done